### PR TITLE
:gear: Fix 'make test.radiusd-c'

### PR DESCRIPTION
**Pull request by jpereira**
It fixes the file stamp related to that test.

### CODE REVIEW

**Comment by jpereira** [comment]
```
[jpereira@jorge-sugarloaf freeradius-server.git]$ make test.radiusd-c
radiusd -C... Current state of talloced memory:
ok
[jpereira@jorge-sugarloaf freeradius-server.git]$
[jpereira@jorge-sugarloaf freeradius-server.git]$
[jpereira@jorge-sugarloaf freeradius-server.git]$ make test.radiusd-c
[jpereira@jorge-sugarloaf freeradius-server.git]$
```

File: src/tests/all.mk [hunk 1]
```diff
@@ -20,7 +20,8 @@ raddb/test.conf:
 #
 # Only redirect STDOUT, which should contain details of why the test failed.
 # Don't molest STDERR as this may be used to receive output from a debugger.
-radiusd-c $(BUILD_DIR)/tests/radiusd-c: raddb/test.conf ${BUILD_DIR}/bin/radiusd $(GENERATED_CERT_FILES) | $(BUILD_DIR)/tests build.raddb
+.PHONY:
```
Comment (alandekok) [inline src/tests/all.mk:23]: I don't think an empty `.PHONY:` does anything.  And the rule `$(BUILD_DIR)/tests/radiusd-c` creates an actual file, so it shouldn't be `.PHONY:`

File: src/tests/all.mk [hunk 2]
```diff
@@ -33,6 +34,9 @@ radiusd-c $(BUILD_DIR)/tests/radiusd-c: raddb/test.conf ${BUILD_DIR}/bin/radiusd
 	@echo "ok"
 	${Q}touch $@
 
+.PHONY:
```
Comment (alandekok) [inline src/tests/all.mk:37]: this should be `.PHONY: test.radiusd-c`

Comment (jpereira) [reply src/tests/all.mk]: typo! :(

fixed/updated.

**Pull request merged**